elastic/kibana
# [AO] Sync chart pointers on the metric threshold alert details page

Closes #155354

## Summary

This PR syncs chart pointers on the metric threshold alert details page.

![image](https://user-images.githubusercontent.com/12370520/233380782-bf97eab8-167d-4f97-a10d-d2bfec1936e7.png)


## 🧪 How to test

- Add `xpack.observability.unsafe.alertDetails.metrics.enabled: true` to the Kibana config
- Create a metric threshold rule with multiple conditions that generates an alert
- Go to the alert details page and check the chart pointers
